.builders/generate_lock.py

```diff
@@ -23,7 +23,6 @@
     'linux-aarch64': 'manylinux.*_aarch64|linux_aarch64',
     'windows-x86_64': 'win_amd64',
     'macos-x86_64': 'macosx.*_(x86_64|intel|universal2)',
-    'macos-aarch64': 'macosx.*_(aarch64|arm64|universal2)',
```

Comment:
Kindly don't remove this line otherwise we won't be able to build the agent for macOS on AArch64/AMD64, see:
https://github.com/DataDog/datadog-agent/blob/fc93bfca4fd61f3a4234fed246234360ac1369a3/omnibus/config/software/datadog-agent-integrations-py3.rb#L89

Reply:
I thought we merged master to incorporate your changes - apologies.